### ISSUE TITLE
Saving model (Based on accuracy)

### ISSUE THREAD

**Issue by Freemanlabs** [deepinsight/insightface#2233]
Hi all,

Working with this codebase it looks like it doesn't check verification (or any form of) accuracy before saving the model like most deep learning implementation. Why is that? Isn't there a possibility of overfitting?

I feel like model that doesn't perform well as the previous ones might be saved thereby leading to a poorer performance when validation on a different dataset.

Please clarify/explain.

Thanks

**Comment by nttstar** [comment]
there's no clear validation set in face recognition problem.
